ts/sagas/index.ts

```diff
@@ -29,15 +30,16 @@ const connectionMonitorParameters = {
 export default function* root(): Iterator<Effect> {
   yield all([
     fork(authenticationSaga),
+    fork(backendInfoSaga),
```

Comment:
why the reordering?

Reply:
I reorder them alphabetically, just as the imported modules are.

Reply:
the order has a meaning as sagas gets started in that order, please revert to the previous order :)

Reply:
Oh, ok. Are they not executed asynchronously in "parallel"?

Reply:
yes but they get started [in sequence one after the other](https://github.com/redux-saga/redux-saga/blob/b3a060a0d67942f0996c80059181352fb1526958/packages/core/src/internal/proc.js#L578)